packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/users.controller.js

```diff
@@ -19,6 +19,15 @@ export default class PciStoragesContainersUsersController {
 
   $onInit() {
     this.loadMessages();
+    this.convertDate();
+  }
+
+  convertDate() {
+    this.userList = this.userList.map((user) => ({
+      ...user,
+      gridPropertyCreationDate: new Date(user.creationDate),
```

Comment:
**question:** instead of adding a new property, can we just rewrite the existing `creationDate`?

Reply:
I prefer to create another attribute since the date is in different format